manim/mobject/coordinate_systems.py

```diff
@@ -740,13 +755,19 @@ def construct(self):
                     0
                 ],
                 target=x,
-                lower_bound=self.x_range[0],
-                upper_bound=self.x_range[1],
+                lower_bound=0,
+                upper_bound=1,
             )
             if alpha is not None:
                 return graph.point_from_proportion(alpha)
             else:
-                return None
+                raise ValueError(
+                    "x={} not located in the range of the graph ([{}, {}])".format(
+                        x,
+                        self.p2c(graph.get_start())[0],
+                        self.p2c(graph.get_end())[0],
```

Comment:
Actually I missed one thing😅:

nitpicky, but I think this should be an f-string to be more consistent with the rest of the library.

Reply:
You mean something like that?
```py
f"x={x} not located in the range of the graph ([{self.p2c(graph.get_start())[0]}, {self.p2c(graph.get_start())[0]}])" 
```

Reply:
Yup, like that👍

Reply:
This is fixed!